python/lsst/cp/pipe/ptc/cpPtcSolve.py

```diff
@@ -792,8 +795,9 @@ def funcFullCovarianceModelNoB(self, params, x):
         matrixSideFit = self.config.maximumRangeCovariancesAstierFullCovFit
         lenParams = matrixSideFit*matrixSideFit
         aMatrix = params[:lenParams].reshape((matrixSideFit, matrixSideFit))
-        cMatrix = params[lenParams:2*lenParams].reshape((matrixSideFit, matrixSideFit))
-        noiseMatrix = params[2*lenParams:3*lenParams].reshape((matrixSideFit, matrixSideFit))
+        cMatrix = np.zeros_like(aMatrix)
+        # params[lenParams:2*lenParams].reshape((matrixSideFit, matrixSideFit))
```

Comment:
What is this commented line?

Reply:
Oh, just forgot to delete this line. I removed the cMatrrix parameter for funcFullCovarianceModelNoB and replaced it with cMatrix = np.zeros_like(aMatrix) internally.